pymeasure/instruments/mock.py

```diff
@@ -49,6 +49,13 @@ def __init__(self, wait=.1, **kwargs):
                        'output_voltage': 'V',
                        'time': 's',
                        'wave': 'a.u.'}
+        self.running = False
```

Comment:
Are lines 52-58 here needed for all instruments with your InstrumentControlWidget? If so, a better place for this would probably be the Instrument base class.

Reply:
No, I just added this to have an example for functions implemented in an Instrument class and to see that the function actually works.